randovania/data/json_data/prime1.json

```diff
@@ -171,7 +171,7 @@
                 "extra": null
             },
             {
-                "index": 24,
+                "index": 42,
```

Comment:
24 is the correct value for Prime 1. Gotta adjust the rest of the code to handle having different ids depending on the game, which is a pending thing still for these resource_database checks

Reply:
Should be fixed with the additions to resource_database